debian/droidian-quirks-device.droidian-boot-wlan.service

```diff
@@ -1,14 +1,13 @@
 [Unit]
-Description=Enables WLAN on devices with boot_wlan
-Before=lxc@android.service
-Requires=android-mount.service
-After=android-mount.service
-ConditionPathExists=/sys/kernel/boot_wlan/boot_wlan
```

Comment:
I liked specifying all the conditions here, I think it makes sense as you did in the older revision.

For the other PR (LMK) I was unconfortable in putting conditions since that script may evolve in something more complex in future - I don't see this being the case here though

Reply:
Ah oki. I was thinking that since every check is done in the script they shouldn't be also here. Will fix when I get at home